src/Lime.Messaging/Resources/ContactDocument.cs

```diff
@@ -128,6 +130,18 @@ protected ContactDocument(MediaType mediaType)
         /// </summary>
         [DataMember(Name = LAST_NAME_KEY)]
         public string LastName { get; set; }
+
+        /// <summary>
+        /// The contact birth date following ISO 8601.
+        /// </summary>
+        [DataMember(Name = BIRTH_DATE_KEY)]
+        public string BirthDate { get; set; }
```

Comment:
Use DateTime instead.

```suggestion
        public DateTime? BirthDate { get; set; }
```